### PR TITLE
add chapter model

### PR DESCRIPTION
**Pull request by Naphtali-cpu**

### CODE REVIEW

File: app/controllers/countries_controller.rb [hunk 1]
```diff
@@ -51,8 +51,8 @@ def update
 
   # DELETE /countries/1 or /countries/1.json
   def destroy
+    @country.chapters.destroy_all # Here we delete associated chapters
```
Comment (kamalogudah) [inline app/controllers/countries_controller.rb:54]: We can get rid of this line and  add    `has_many :chapters, dependent: :destroy` on the country  model

File: app/models/country.rb [hunk 2]
```diff
@@ -11,4 +11,7 @@ class Country < ApplicationRecord
 
   # Validations
   validates :name, presence: true
+
+  #Associations
+  has_many :chapters
```
Comment (kamalogudah) [inline app/models/country.rb:16]: `has_many :chapters, dependent: :destroy`  as suggested above

**Comment by kamalogudah** [comment]
Good work  just a few comments. While at it could  you also add some  tests  https://guides.rubyonrails.org/testing.html  we  will stick with  minitest,the official tests  by rails.

File: app/controllers/countries_controller.rb (hunk 1 above)
Comment (Naphtali-cpu) [reply app/controllers/countries_controller.rb:54]: Yeah, let me make the changes after going through that link.

File: app/models/country.rb (hunk 2 above)
Comment (Naphtali-cpu) [reply app/models/country.rb:16]: Noted

**Pull request closed without merging**